server/embed/serve.go

```diff
@@ -191,6 +192,11 @@ func (sctx *serveCtx) serve(
 			TLSConfig: tlscfg,
 			ErrorLog:  logger, // do not log user error
 		}
+		if err := http2.ConfigureServer(srv, &http2.Server{
+			MaxConcurrentStreams: v3rpc.MaxStreams,
+		}); err != nil {
+			return err
+		}
```

Comment:
move the code at the beginning of the function, so that it can take effect for both http and https.

Reply:
> move the code to the beginning of the function, so that it can take effect for both http and https.
the http server always work good,it haved impacted by the MaxConcurrentStreams settings